x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -231,6 +233,32 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 			details["rewardedCU"] = strconv.FormatInt(reward.Quo(coinsPerCu).TruncateInt64(), 10)
 		}
 
+		// track the provider's CU (after QoS influence)
```

Comment:
can we add this to the chargeCU method?
we already fetch the project and subscription there

Reply:
done. see https://github.com/lavanet/lava/commit/ddf3113fc8632e5dea14e5825f1f26c914758a13